db/seeds.rb

```diff
@@ -5,5 +5,21 @@
 #
 #   movies = Movie.create([{ name: 'Star Wars' }, { name: 'Lord of the Rings' }])
 #   Character.create(name: 'Luke', movie: movies.first)
-puts "Importing Schools information, this may take a minute..."
-SchoolDataImporter.new.run
+
+unless Rails.env.development?
+  raise "This db/seeds.rb file is only intended for use in a local development environment. It should not be used in the #{Rails.env} environment."
+end
+
+ENV["FIXTURES_PATH"] = "spec/fixtures"
+Rake::Task["db:fixtures:load"].invoke
+
+class Seeds
+  extend FactoryBot::Syntax::Methods
+
+  create(:payroll_run, :confirmation_report_uploaded, claims_count: 32, created_at: 3.months.ago - 10.days)
```

Comment:
If you didn't want to extend `FactoryBot::Syntax::Methods` you can just statically call the factory helpers:

```ruby
FactoryBot.create(:payroll_run, :confirmation_report_uploaded, claims_count: 32, created_at: 3.months.ago - 10.days)
```
I think either is fine though

Reply:
Yeah, I don't really have any strong opinions either way; thought this was just a little more readable.